apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component, PropTypes } from 'react';
+import _ from 'lodash';
+import { assignmentShape } from './shapes';
+
+export default class AssignmentSelector extends Component {
+  static propTypes = {
+    courseId: PropTypes.number,
+    scriptId: PropTypes.number,
+    validCourses: PropTypes.arrayOf(assignmentShape).isRequired,
+    validScripts: PropTypes.arrayOf(assignmentShape).isRequired,
+  };
+
+  getSelectedAssignment() {
+    const assignment = this.assignments[this.root.value];
+    return {
+      course_id: assignment.course_id,
+      script_id: assignment.script_id
+    };
+  }
```

Comment:
This isn't actually used right now. The idea is that when we save, we will extra the values from each of our fields in the SectionRow, construct a section object, and then POST an update to the server. This particular method is our way of figuring out course_id/section_id based on the current value in our dropdown.